teacher.rb

```diff
@@ -0,0 +1,16 @@
+require_relative 'person'
+
+class Teacher < Person
+  def initialize(age, specialization, name = 'Unknown', parent_permission: true)
+    super(age, name, parent_permission: parent_permission)
+    @specialization = specialization
+  end
+
+  def can_use_services?
+    true
+  end
+end
+
+teacher1 = Teacher.new(12, 'Math')
+p 'CLASS TRACHER'
+p teacher1.can_use_services?
```

Comment:
- [ ] [Optional][highly recommended] Same here, It would be better professionally if you added all test cases for all modules in a separate file.